aries/aries_vcx_core/src/wallet/entry_tags.rs

```diff
@@ -2,32 +2,26 @@ use std::fmt;
 
 use serde::{de::Visitor, ser::SerializeMap, Deserialize, Serialize};
 
-#[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord)]
-pub enum EntryTag {
-    Encrypted(String, String),
-    Plaintext(String, String),
-}
-
-impl EntryTag {
-    pub fn from_pair(pair: (String, String)) -> Self {
-        if pair.0.starts_with('~') {
-            EntryTag::Plaintext(pair.0.trim_start_matches('~').into(), pair.1)
-        } else {
-            EntryTag::Encrypted(pair.0, pair.1)
-        }
-    }
-
-    pub fn key(&self) -> &str {
-        match self {
-            Self::Encrypted(key, _) => key,
-            Self::Plaintext(key, _) => key,
-        }
-    }
-}
+// #[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord)]
+// pub enum EntryTag {
+//     Tag(String, String),
+// }
+
+// impl EntryTag {
+//     pub fn from_pair(pair: (String, String)) -> Self {
+//         Self::Tag(pair.0, pair.1)
+//     }
+
+//     pub fn key(&self) -> &str {
+//         match self {
+//             Self::Tag(key, _) => key,
+//         }
+//     }
+// }
 
 #[derive(Debug, Default, Clone, PartialEq)]
 pub struct EntryTags {
-    inner: Vec<EntryTag>,
+    inner: Vec<(String, String)>,
```

Comment:
Perhaps we can keep
```rust
pub type EntryTag = (String, String);
```
That way, the type is defined one place instead of being scattered around.